lighthouse-core/gather/driver/network-monitor.js

```diff
@@ -90,7 +90,10 @@ class NetworkMonitor {
     this._frameNavigations = [];
     this._sessions = new Map();
     this._networkRecorder = new NetworkRecorder();
-    this._targetManager = new TargetManager(this._session);
+    /** @type {LH.Puppeteer.CDPSession} */
+    // @ts-expect-error - temporarily reach in to get CDPSession
+    const rootCdpSession = this._session._session;
+    this._targetManager = new TargetManager(rootCdpSession);
```

Comment:
a required hack (we need the underlying CdpSession but the FRProtocolSession doesn't include that). A follow up change can move `target-manager` from inside `network-monitor` to e.g. `driver` (and passed in to `network-monitor` to maintain the current use), so the CdpSession will be readily available then. Cut off here to keep the PR simpler.